devfile.yaml

```diff
@@ -27,14 +23,14 @@ components:
       - '-c'
       - >-
         while [ ! -f
-        ${CHE_PROJECTS_ROOT}/src/github.com/devfile/kubernetes-api/.theia/settings.json
-        -o  ! -f ${CHE_PROJECTS_ROOT}/.theia/settings.json ]; do echo "Waiting
-        for the clone end" ; sleep 2; done; sleep 2 ; echo "Updating the yaml
-        schema bindings" ; sed -e
+        ${CHE_PROJECTS_ROOT}/src/github.com/devfile/kubernetes-api/.theia/settings.json ];
```

Comment:
Can we remove this busy box container by moving this command to tools container?